src/styles/pages/_user-view.scss

```diff
@@ -0,0 +1,12 @@
+.navbar-toggle {
+  border: none !important;
```

Comment:
Ideally we'll keep specificity flat. If that's not possible, you can use the same class selector multiple times to up the specificity

`.navbar-toggle.navbar-toggle`